app/src/api/dataAccess/__mocks__/editHistory.ts

```diff
@@ -0,0 +1,62 @@
+import { EditHistoryEntry } from '../../../frontend/models/edit-history-entry';
+import { numAsc, numDesc } from '../../../helpers';
+
+/**
+ * Create an object mocking all method of editHistory dataAccess
+ * The type is set to reflect the type of that module, with added methods
+ * used when testing
+ */
+const mockEditHistory =
+    jest.genMockFromModule('../editHistory') as typeof import('../editHistory') & {
+        __setHistory: (mockHistoryData: EditHistoryEntry[]) => void
+    };
```

Comment:
Setting the type to that of the original mocked module, and extending it with a local type containing the `__setHistory` method, allows for checking that the method mocks in this file align with the definitions in the original module, while allowing to use special methods needed to set up the mock during testing.